### PR TITLE
add nativescript workflows

### PR DESCRIPTION
**Pull request by erodriguezh**
Hi Workflow Team,

I added the workflows for NativeScript.

Best

### CODE REVIEW

**Comment by elviskahoro** [comment]
@erodriguezh I'm gonna add aloke as a reviewer. Thanks again for opening a PR and adding native script! You've added a lot of workflows, we appreciate it a lot!

**Comment by elviskahoro** [comment]
@erodriguezh if you're in our discord what's your username so I can add the contributor discord role?

**Comment by erodriguezh** [comment]
@elviskahoro my username is erodr#3149, I'm not sure if the hash number is necessary 😅

**Review by ianhodge — CHANGES_REQUESTED** [review]
Hey @erodriguezh ! Thank you SO much for creating all these workflows. This is super beneficial to the whole Warp community. 

One small note - right now the title and descriptions of these workflows are a little vague and I think should be a little more specific to nativescript. For instance the unit test workflow "Configures your project for unit testing with a selected framework" could be "Configures your NativeScript project for unit testing with a selected framework." This way, users who are not searching specifically for a NativeScript command won't be confused by the contents of the workflow, and users who want to search for a NativeScript command can find it more easily. 

Let me know if you have any questions and thank you so much again for contributing! This is really amazing.

**Comment by erodriguezh** [comment]
Hi @ianhodge,

Thanks for your feedback, it is a good point. Please check the latest commit, I added accuracy to the titles and descriptions.

**Review by ianhodge — APPROVED** [review]
Looks great! Thank you for addressing the feedback :)

**Comment by erodriguezh** [comment]
There seemed to be an issue with double quotes, should be ok now.

File: specs/nativescript/add_platform.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Configure your current NativeScript project to target the selected platform.
+command: "ns platform add {{platform}}"
+tags:
+  - ns
+  - tns
```
Comment (alokedesai) [inline specs/nativescript/add_platform.yaml:5]: Hey @erodriguezh! Thanks so much for your contribution. One thing i'm noticing is that we have duplicated tags for `ns`, `tns`, and `nativescript` here. Would it be possible to consolidate and just have the `nativescript` tag instead?

**Pull request merged**